fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/MetaDataManager.java

```diff
@@ -67,11 +71,41 @@ public void createDatabase(String databaseName, boolean ignoreIfExists)
                     "Database " + databaseName + " already exists.");
         }
 
+        DatabaseRegistration databaseRegistration = DatabaseRegistration.of(databaseDescriptor);
+
         uncheck(
-                () -> zookeeperClient.registerDatabase(databaseName),
+                () -> zookeeperClient.registerDatabase(databaseName, databaseRegistration),
                 "Fail to create database: " + databaseName);
     }
 
+    public DatabaseInfo getDatabase(String databaseName) throws DatabaseNotExistException {
+        if (!databaseExists(databaseName)) {
+            throw new DatabaseNotExistException("Database " + databaseName + " does not exist.");
+        }
+
+        Optional<DatabaseRegistration> optionalTable;
+        try {
+            optionalTable = zookeeperClient.getDatabase(databaseName);
+        } catch (Exception e) {
+            throw new FlussRuntimeException(
+                    String.format("Fail to get database '%s'.", databaseName), e);
+        }
+
+        DatabaseRegistration databaseReg = null;
+        if (optionalTable.isPresent()) {
+            databaseReg = optionalTable.get();
+        } else {
+            // the database node before 0.6 is empty.
+            databaseReg = DatabaseRegistration.of(DatabaseDescriptor.builder().build());
```

Comment:
If the node is empty, it means the database is not exist, we should throw `DatabaseNotExistException`? And we don't need to check `databaseExist` at the beginning of this method. (just similar to the implementation of `getTable`. )